src/components/SelectionListRadio/BaseSelectionListRadio.js

```diff
@@ -254,7 +255,13 @@ function BaseSelectionListRadio(props) {
                             maxToRenderPerBatch={5}
                             windowSize={5}
                             viewabilityConfig={{viewAreaCoveragePercentThreshold: 95}}
-                            onLayout={() => scrollToIndex(focusedIndex, false)}
+                            onLayout={() => {
+                                if (!firstLayoutRef.current) {
+                                    return;
+                                }
```

Comment:
Another option would have been to have a state + useEffect:

```
const [onLayoutTriggered, setOnLayoutTriggered] = useState(false);
useEffect(() => if (onLayoutTriggered) scrollToIndex(focusedIndex, false), [focusedIndex, onLayoutTriggered]);


...
<SectionList
       onLayout={() => setOnLayoutTriggered(true)}
</SectionList>
```

but I'm not really sure that that is better than what you did with the ref.

Reply:
> Hm... Are you sure about this? I mean it would be better if the scrollToIndex function would have single responsibility.

Agree, I prefer how it is now.